src/ArgentPonyWarcraftClient.Extensions.DependencyInjection/ArgentPonyWarcraftClient.Extensions.DependencyInjection.csproj

```diff
@@ -0,0 +1,36 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <Authors>Dan Jagnow, Travis Boatman, William Wolfram</Authors>
+    <Description>Extensions of Microsoft.Extensions.DependencyInjection for the Argent Pony .NET client.</Description>
+    <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
+    <Copyright>Copyright © Dan Jagnow 2017</Copyright>
+    <NeutralLanguage>en-US</NeutralLanguage>
+    <Title>Argent Pony Warcraft Client</Title>
+    <PackageTags>Warcraft World-of-Warcraft WoW Blizzard Dependency-Injection DI</PackageTags>
```

Comment:
Noticed the documentation indicates that the `<PackageTags>` element is supposed to be a semi-colon delimited list: https://docs.microsoft.com/en-us/dotnet/core/tools/csproj#packagetags

Reply:
Ah, nice catch.  I'll take care of that in a follow-up change.

Reply:
The follow-up change is https://github.com/blizzard-net/warcraft/pull/144.